app/src/common/shared/com/igalia/wolvic/ui/widgets/NavigationBarWidget.java

```diff
@@ -1175,12 +1175,23 @@ public void onSharedPreferenceChanged(@NonNull SharedPreferences sharedPreferenc
         }
     }
 
+    private boolean isVRProjection(int projection){
+        switch (projection) {
+            case VIDEO_PROJECTION_NONE:
```

Comment:
This was not in the previous code so I wouldn't include it although I understand that the cases are similar (front facing videos).

Regarding `isVRProjection` I'd look for another name as 360 or 180 are also VR projections. Perhaps we should focus on the use case, something like `isFrontFacingVRProjection()`

Reply:
Done!